Chrysalis-Papilio/Soen-490-Papilio-Android
# [PAPILIO-14] Activity page

# General info

**Issue number**: PAPILIO-14

**Task description**: 
Created a Feed page where activities will be shown to the user based on their preferences

# Testing
To test this feature you have to:

1. Open the Papilio app
2. Log in or sign up. Either way, you will be redirected to the home page where the first thing you'll see is the feed
3. Verify that you can see a list of activities and that you can scroll them
4. Verify that more activities load the further you scroll down
5. Verify that when you tap on a single activity, it takes you to the info page for that specific page
6. On the info page for that activity, verify that you can see all the important information related to the activity including the images, the map and reviews. 
7. Verify that the images are scrollable
8. If steps 3 to 7 are verified successfully, then this story this hereby complete.
